pkg/config/config.go

```diff
@@ -71,6 +71,8 @@ type CniConfig struct {
 	NetworkPluginBinDir string `toml:"bin_dir" json:"binDir"`
 	// NetworkPluginConfDir is the directory in which the admin places a CNI conf.
 	NetworkPluginConfDir string `toml:"conf_dir" json:"confDir"`
+	// NetworkPluginMaxConfNum is the max num of plugin file will load
```

Comment:
s/max num of plugin file/max number of plugin config file

Reply:
Can we explain what this number is and how it is used?

Reply:
Can you link me to the value is used pls?

Reply:
Ha, yeah we should update docs/config.md.

Reply:
@Random-Liu Yeah, that's right. 
@mikebrow In our case, we add a second NIC of the pod, and set a LAN ip for it.The second NIC is Optional，and we control the ips for every deployment stable.

Reply:
Ok, interesting use case... 

I don't see code in cni proper for using this value.. but I may have missed it? Is this for a custom plugin or a to be dropped pr for cni? Is this value just a pass through value? Is there a mockup of this pattern somewhere? Inquiring minds want to know :-)

Reply:
The value is used in `go-cni` library. :)

Reply:
@kuramal Can you update https://github.com/containerd/cri/blob/master/docs/config.md, and include some explanation of the option, like https://github.com/containerd/cri/pull/1155#discussion_r291023199?

Reply:
ok found it in go-cni/opts.go 

// loadFromConfDir detects network config files from the
// configured cni config directory and load them. max is
// the maximum network config to load (max i<= 0 means no limit).

Reply:
@Random-Liu  Done